openapi-v3.yaml

```diff
@@ -147,6 +142,56 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  /accounts/{accountId}/users:
+    get:
+      tags:
+        - Accounts
+      summary: List Users
+      description: Use the List Users endpoint to list the Users in the specified Account
+      operationId: listUsers
+      parameters:
+        - $ref: '#/components/parameters/accountId'
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UsersResponse'
+  /accounts/{accountId}/permissions/{licenseId}:
+    post:
+      tags:
+        - Accounts
+      summary: Update License
+      description: Update (or deactivate) permissions for a given license
+      operationId: updateLicense
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/License'
+      parameters:
+        - $ref: '#/components/parameters/accountId'
+        - in: path
+          name: licenseId
+          schema:
+            type: integer
+          required: true
+          description: Numeric ID of the License to update
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UpdateLicenseResponse'
+        '404':
+          description: Unauthorized or Not Found.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
```

Comment:
This is in the postman collection under how to [deactivate a license](https://documenter.getpostman.com/view/14183654/UVsSNiXC#ba3667e8-c61d-418c-88ea-a889051184f0) - how much detail (if any) do we want to go into here?

Reply:
I think this looks good - but maybe @jeremyhutt11 or the SAs have additional thoughts on what they'd want to see documented?